### PR TITLE
Cleanup rocksdb config and metrics

### LINKED ISSUE #6574: Add missing db size metrics and make sure each one works properly.
I've noticed that our metrics do not include any blob_transaction db size information.
Current size for goerli after Dencun:
```
59G	    ./sedge-data/execution-data/nethermind_db/goerli/state
45G	    ./sedge-data/execution-data/nethermind_db/goerli/receipts
256G	./sedge-data/execution-data/nethermind_db/goerli/blocks
8.9G	./sedge-data/execution-data/nethermind_db/goerli/code
4.0K	./sedge-data/execution-data/nethermind_db/goerli/peers
4.1M	./sedge-data/execution-data/nethermind_db/goerli/witness
461M	./sedge-data/execution-data/nethermind_db/goerli/blockInfos
550M	./sedge-data/execution-data/nethermind_db/goerli/blockNumbers
4.1M	./sedge-data/execution-data/nethermind_db/goerli/badBlocks
2.7G	./sedge-data/execution-data/nethermind_db/goerli/bloom
3.4G	./sedge-data/execution-data/nethermind_db/goerli/headers
4.1M	./sedge-data/execution-data/nethermind_db/goerli/canonicalHashTrie
4.2M	./sedge-data/execution-data/nethermind_db/goerli/metadata
1.8M	./sedge-data/execution-data/nethermind_db/goerli/discoveryNodes
75M	    ./sedge-data/execution-data/nethermind_db/goerli/blobTransactions
376G	./sedge-data/execution-data/nethermind_db/goerli
376G	./sedge-data/execution-data/nethermind_db
```

vs currently visible metrics in Grafana:
```
nethermind_cht_db_size
nethermind_code_db_size
nethermind_bloom_db_size
nethermind_state_db_size
nethermind_blocks_db_size
nethermind_headers_db_size
nethermind_witness_db_size
nethermind_receipts_db_size
nethermind_metadata_db_size
nethermind_block_infos_db_size
```

So I see missing:
```
blobTransactions
discoveryNodes
badBlocks
blockNumbers
peers
```

Additionally, I see that these metrics are reporting invalid data comparing to actual nethermind_Db content:
- Receipts_db: In Goerli it is 45GB, metric shows 100KB
- Metadata_db: in Goerli it is 4.2 MB, mertric shows 0
- Witness_db: in Goerli it shows 4.1MB, metric shows 0
- Bloom_db: in Goerli it is 2.7GB, metric shows 0
- Canonical Hash Tree_db: in Goerli it shows 4.1MB, metric shows 0

### PR DESCRIPTION
**Pull request by asdacap**
Fixes #6574 

## Changes

- Exposes more rocksdb config and added comments on them. No config is actually changed.
- Change the estimate measurement for heavywrite tune from l0 file num to l0 layer size. No config actually changed, just how its calculated.
- Exposes rocksdb statistics as metrics. This metrics contain things such as cache hit, bloom usefulness, read bytes, blob file read bytes, etc... there is quite a lot in it actually. Of course, it is not served by default.
- Fix database size, cache size and other db size not working properly for column db.
- Fix column db not using same cache as main column in the same column db.
- Use label instead of separate metric for each separate db so that we don't miss db metric anymore.

#### What types of changes does your code introduce?

- [X] Bugfix (a non-breaking change that fixes an issue)
- [X] New feature (a non-breaking change that adds functionality)
- [X] Refactoring

#### Requires testing

- [X] Yes
- [ ] No

#### If yes, did you write tests?

- [X] Yes
- [ ] No

#### Notes on testing

- Seems to work. 
- [X] Db block cache size
- [X] Db memtable size
- [X] Db index filter size
- [X] Db size
- [X] Db read
- [X] Db write

## Remarks

- Please use label. No need to sum at application level. Prometheus/grafana can sum by label, node, etcs.

### CODE REVIEW

File: src/Nethermind/Nethermind.Db.Rocks/Statistics/DbMetricsUpdater.cs [hunk 1]
```diff
@@ -67,6 +68,42 @@ private void UpdateMetrics(object? state)
...
+            // The metric can be of several type, usually just a counter, but sometime its a histogram,
+            // in which case we take both the sum and count.
+            if (value.SubMetric.TryGetValue("SUM", out var valueSum))
+            {
+                Metrics.DbStats[(_dbName + "Db", value.Name + ".sum")] = valueSum;
+                Metrics.DbStats[(_dbName + "Db", value.Name + ".count")] = value.SubMetric["COUNT"];
+            }
+            else
+            {
+                Metrics.DbStats[(_dbName + "Db", value.Name)] = value.SubMetric["COUNT"];
```
Comment (rubo) [inline src/Nethermind/Nethermind.Db.Rocks/Statistics/DbMetricsUpdater.cs:84]:
```suggestion
                Metrics.DbStats[($"{_dbName}Db", value.Name)] = value.SubMetric["COUNT"];
```

File: src/Nethermind/Nethermind.Db/Metrics.cs [hunk 2]
```diff
@@ -11,202 +11,233 @@ namespace Nethermind.Db
     public static class Metrics
     {
         [CounterMetric]
-        [Description("Number of Bloom DB reads.")]
+        [Description("[DEPRECATED] Number of Bloom DB reads.")]
```
Comment (rubo) [inline src/Nethermind/Nethermind.Db/Metrics.cs:14]:
```suggestion
        [Description("_Deprecated._ Number of Bloom DB reads.")]
```

Please replace it like this below to be consistent with the docs styling.

Comment (asdacap) [reply src/Nethermind/Nethermind.Db/Metrics.cs]: Done

File: src/Nethermind/Nethermind.Db.Rocks/Statistics/DbMetricsUpdater.cs [hunk 3]
```diff
@@ -67,6 +68,42 @@ private void UpdateMetrics(object? state)
...
+    public void ProcessStatisticsString(string dbStatsString)
+    {
+        foreach ((string Name, IDictionary<string, double> SubMetric) value in ExtractStatsFromStatisticString(dbStatsString))
+        {
+            // The metric can be of several type, usually just a counter, but sometime its a histogram,
+            // in which case we take both the sum and count.
+            if (value.SubMetric.TryGetValue("SUM", out var valueSum))
+            {
+                Metrics.DbStats[(_dbName + "Db", value.Name + ".sum")] = valueSum;
+                Metrics.DbStats[(_dbName + "Db", value.Name + ".count")] = value.SubMetric["COUNT"];
```
[the remark this answers is not in the document]
Comment (asdacap) [reply src/Nethermind/Nethermind.Db.Rocks/Statistics/DbMetricsUpdater.cs]: Done

File: src/Nethermind/Nethermind.Db.Rocks/Statistics/DbMetricsUpdater.cs (hunk 1 above)
Comment (asdacap) [reply src/Nethermind/Nethermind.Db.Rocks/Statistics/DbMetricsUpdater.cs]: Done

**Review by rubo — APPROVED** [review]

**Pull request merged**